master/internal/config/config.go

```diff
@@ -475,6 +485,12 @@ func (t *TLSConfig) Validate() []error {
 // Validate implements the check.Validatable interface.
 func (t *SSHConfig) Validate() []error {
 	var errs []error
+	if t.KeyType != RSAKeyType && t.KeyType != ECDSAKeyType && t.KeyType != ED25519KeyType {
+		errs = append(errs, errors.New("Crypto system must be one of 'RSA', 'ECDSA' or 'ED25519'"))
+	}
+	if t.KeyType != RSAKeyType {
```

Comment:
can we flip this and put this rsa specific stuff in an `if t.KeyType == RSAKeyType` block? written like this, it's easy for someone to add a check and the bottom of this func and not realize it's just for RSA.

Reply:
+1.